src/Symfony/Component/Scheduler/RecurringMessage.php

```diff
@@ -46,6 +46,19 @@ public static function cron(string $expression, object $message): self
         return new self(CronExpressionTrigger::fromSpec($expression), $message);
     }
 
+    public static function hashedCron(string $expression, object $message, string $context = null): self
```

Comment:
We could merge this method into `cron()`:
- detect if a hash is being used (expression contains `#`)
- add optional `$context` parameter